core/src/main/scala/kafka/server/checkpoints/OffsetCheckpointFile.scala

```diff
@@ -62,12 +62,17 @@ trait OffsetCheckpoint {
  *  -----checkpoint file end----------
  */
 class OffsetCheckpointFile(val file: File, logDirFailureChannel: LogDirFailureChannel = null) {
-  val checkpoint = new CheckpointFileWithFailureHandler[(TopicPartition, Long)](file, OffsetCheckpointFile.CurrentVersion,
+  val checkpoint = new CheckpointFileWithFailureHandler[Tuple2[TopicPartition, Long]](file, OffsetCheckpointFile.CurrentVersion,
```

Comment:
What is the reason for this change?

Reply:
Nice catch! This is not needed, it seemed to be part of earlier refactoring.